src/lib/modules/blockchain_process/blockchain.js

```diff
@@ -146,14 +146,17 @@ Blockchain.prototype.initProxy = function () {
 };
 
 Blockchain.prototype.setupProxy = async function () {
+  const AccountParser = require('../../utils/accountParser');
```

Comment:
Can we move this require statement out of the function? Ideally we refactor this straight to `import * as AccountParser` ..

Actually it'd be nicer if we could do `import { AccountParser } from '...` but we'll have to refactor the exports of AccountParser for that.